.ci/resolve-kata-dependencies.sh

```diff
@@ -19,6 +18,14 @@ pr_branch=${pr_branch:-}
 pr_number=${pr_number:-}
 kata_repo=${kata_repo:-}
 
+# Name of the label, that if set on a PR, will ignore depends-on lines in commits
+ignore_depends_on_label="ignore-depends-on"
+
+script_name="${0##*/}"
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+source /etc/os-release || source /usr/lib/os-release
```

Comment:
I know we're not exactly minimal in a bunch of places, but are we using `os-release` for anything here?

Reply:
Agreed, I don't use it, I probably get copied too many lines over. I've removed this line now. Thanks!